docs/src/reference/the-traversal.asciidoc

```diff
@@ -1236,8 +1236,8 @@ g.V()
 [[e-step]]
 === E Step
 
-The `E()`-step is meant to read edges from the graph and is usually used to start a `GraphTraversal`. Unlike the related
-<<v-step,V()>> it cannot be used mid-traversal.
+The `E()`-step is meant to read edges from the graph and is usually used to start a `GraphTraversal`, but can also
+be used mid-traversal.
```

Comment:
please add an example of mid-traversal `E()` here

Reply:
added in https://github.com/apache/tinkerpop/pull/1808/commits/13501c50233d1692d2cd3dd8185e61e9050f8fe3